test/controllers/expenses_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ExpensesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
Required
- Please make sure to not leave any empty file or unused code for a cleaner code base.